### ISSUE TITLE
How to disallow '*' type?

### ISSUE THREAD

**Issue by synthecypher** [gajus/eslint-plugin-jsdoc#166]
How can I disallow the `*` type?
```javascript
/**
 * Do something
 * @param {*} a
 * @param {*} b
 * @returns {undefined}
 */
function doSomething(a, b) {
  return undefined;
}
```

**Comment by brettz9** [comment]
This looks like a dupe of #134.

**Comment by brettz9** [comment]
If `preferType` from ESLint's `valid-jsdoc` (#107) is carried over, I'd suggest piggy-backing on this feature so that when keys (i.e., the types) of this object are mapped to values which are `false` (instead of a string indicating a preferred name), then this could indicate a disallowed type.

**Comment by brettz9** [comment]
Closed by #247. See https://github.com/gajus/eslint-plugin-jsdoc/tree/master/.README#settings-to-configure-check-types-and-no-undefined-types for use with `check-types`.

i.e., you'll be able to just add:

```js
{
  settings: {
    jsdoc: {
      preferredTypes: {
        '*': false
     }
  }
}
```
